python/lsst/meas/extensions/scarlet/io.py

```diff
@@ -130,7 +130,7 @@ def asDict(self) -> dict:
         return {
             "xy0": self.xy0,
             "extent": self.extent,
-            "center": self.extent,
+            "center": self.center,
```

Comment:
Because of the mix-n’-match of conventions, it wouldn’t hurt to be explicit about (y, x) vs. (x, y) in the variable doc strings (in addition to your comment in the class description above).  I got nervous looking at your test…!

Reply:
🤪